kratos/tests/test_variable_utils.py

```diff
@@ -693,6 +693,75 @@ def test_apply_vector(self):
             self.assertEqual(node.GetSolutionStepValue(KratosMultiphysics.DISPLACEMENT_X), data_vector_x2[i])
             i+=1
 
+    def test_solutionsteps_vector(self):
+        current_model = KratosMultiphysics.Model()
+
+        ##set the model part
+        model_part = current_model.CreateModelPart("Main")
+        model_part.AddNodalSolutionStepVariable(KratosMultiphysics.VISCOSITY)
+        model_part.AddNodalSolutionStepVariable(KratosMultiphysics.DISPLACEMENT)
+        model_part.AddNodalSolutionStepVariable(KratosMultiphysics.VELOCITY)
+        model_part.CreateNewNode(1,1.0,2.0,3.0)
+        model_part.CreateNewNode(2,11.0,25.0,31.0)
+        model_part.CreateNewNode(10,12.0,26.0,32.0)
+
+        #reference_results
+        ref_initial_coords2d = [1.0,2.0,11.0,25.0,12.0,26.0]

```

Comment:
@loumalouomega i removed the numpy dependency since this test could be done without introducing numpy (i only need lists).
still numpy Is allowed, and should be also present in all of the compile bots.

I saw that you approved, could u please approve again?